src/jobs/database_schema_report.yml

```diff
@@ -45,6 +49,16 @@ parameters:
     default: "42.7.3"
     description: Postgres driver version
 
+executor:
+  name: java_localstack_postgres_with_db_name
+  jdk_tag: << parameters.jdk_tag >>
+  localstack_tag: << parameters.localstack_tag >>
+  postgres_tag: << parameters.postgres_tag >>
+  postgres_db: << parameters.database_name >>
+  postgres_username: << parameters.user >>
+  postgres_password: << parameters.password >>
+  postgres_port: << parameters.port >>
```

Comment:
passing these through to the underlying orb - so same connection details are used on schemaspy and the executor